### DISCUSSION TITLE
Forward node does not update to 2.3.100

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#7198] [Unsupported Versions] [answered]
Hello!

I ran `soup` yesterday to update my distributed deployment and it worked fine for my manager and search nodes, but after 24h my forward node still shows version 2.3.91 in the SOC grid. I already ran `salt-call state.highstate` manually on the forward node without success. Is it safe/necessary to reboot the forward node? `so-status` shows all ok on all nodes.

Cheers, Ben

**Comment by ben-sec** [comment, marked as answer]
After 48h I tried a restart of the forward node. Now `so-status` toggles between [ WAIT_START ] and [ STARTING ] for all docker containers (Docker status itself is OK). So something went wrong with the SO 2.3.100 update. How can I re-run the update on my forward node?

**Reply by ben-sec** [reply]
Update: Can't find any 2.3.100 docker images on my forward node.

**Reply by palichx** [reply]
Hello,
did you install system updates?

**Reply by ben-sec** [reply]
I just ran `soup` on the master node.

**Reply by palichx** [reply]
try to run "yum update" on heavy node.

**Reply by Stephen-Ridgway** [reply]
We also had this issue. For us, the root cause was that one of the yum Google repositories wasn't available, so OS updates hadn't been applied.

The fix was to remove the broken repository, update the os via yum and run a state.highstate via salt on the master

**Reply by ben-sec** [reply]
I can confirm the broken update mirror:

`https://repo.mongodb.org/yum/redhat/7/mongodb-org/4.2/x86_64/repodata/repomd.xml: [Errno 14] curl#22 - "The requested URL returned error: 403"`

I ran yum update  without the mongodb repository and after a `salt-call state.highstate` everything is up and running again. Thanks Stephen!
